source/includes/steps-install-shell-base.yaml

```diff
@@ -23,7 +23,7 @@ content: |
      .. code-block:: sh
 
```

Comment:
[optional suggestion] I know it's not your copy, but in step 2, I feel like we could remove the redundant link to the MongoDB Download Center since we already direct users to it in step 1.

Reply:
Removed!